loginserver/client.cpp

```diff
@@ -583,6 +588,73 @@ void Client::DoSuccessfulLogin(
 	m_client_status = cs_logged_in;
 }
 
+void Client::SendExpansionPacketData(PlayerLoginReply_Struct& plrs)
+{
+	int default_ruleset = 1;
+	SerializeBuffer buf;
+	//from eqlsstr_us.txt id of each expansion, excluding 'Everquest'
+	int ExpansionLookup[20] = { 3007, 3008, 3009, 3010,	3012,
+								3014, 3031, 3033, 3036, 3040,
+								3045, 3046, 3047, 3514, 3516,
+								3518, 3520, 3522, 3524 };
+
+
+	//Get the active expansion from the database Rule:WorldExpansionSettings.  Do this only once.  Requires restart to effect change
+	if (!owned_expansion) {
+		std::string r_name = RuleManager::Instance()->GetRulesetName(server.db, default_ruleset);
+		if (r_name.size() > 0) {
+			RuleManager::Instance()->LoadRules(server.db, r_name.c_str(), false);
+			expansion = RuleManager::Instance()->GetIntRule(RuleManager::Int__ExpansionSettings);
```

Comment:
A config option might make more sense for this since it's loaded once anyway, but I'll defer to others for that

Reply:
Also are rules a standard table for the login server database? Mine doesn't have it but I don't remember the last time I sourced anything so it might just be really out of date

Reply:
Loginserver does not use rules. Configuration should be driven from the config file

Reply:
updated to use login.json instead of rules table.